kyaari-oms/src/dashboards/accounts/pages/Reports.tsx

```
@@ -77,6 +78,103 @@ const PAYMENT_MONTHLY_DATA = [
+    document.body.appendChild(a)
+    a.click()
+    document.body.removeChild(a)
+    URL.revokeObjectURL(url)
+  }
+
+  const toCSV = (rows: Array<Record<string, unknown>>, headerOrder?: string[]) => {
+    if (!rows || rows.length === 0) return ''
+    const headers = headerOrder && headerOrder.length > 0
+      ? headerOrder
+      : Array.from(new Set(rows.flatMap((r) => Object.keys(r))))
+    const escape = (val: unknown) => {
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)